pgraph/inc/WireCellPgraph/Pgrapher.h

```diff
@@ -72,6 +73,7 @@ namespace WireCell {
             virtual WireCell::Configuration default_configuration() const;
         private:
             Graph m_graph;
+            Pgraph::Factory m_fac;
```

Comment:
Seems `m_fac` is not used in the latest commit

Reply:
Thanks.  Latest update removes it.